op-node/node/api.go

```diff
@@ -95,7 +98,9 @@ func (n *nodeAPI) OutputAtBlock(ctx context.Context, number hexutil.Uint64) (*et
 		return nil, fmt.Errorf("failed to get L2 block ref with sync status: %w", err)
 	}
 
-	output, err := n.client.OutputV0AtBlock(ctx, ref.Hash)
+	contextWithTimeout, cancel := context.WithTimeout(ctx, n.rpcTimeout)
+	output, err := n.client.OutputV0AtBlock(contextWithTimeout, ref.Hash)
+	defer cancel()
```

Comment:
I think I'm missing something - this is adding a new timeout using the configured value, but I don't see anywhere in this PR that removes an existing timeout which would be required to make the timeout longer as per the PR description. Currently it looks like the option will only be able to shorten the timeout.

Reply:
Yes! You're right. There's an existing timeout flag in the op-proposer, so we should increase that timeout value. The op-node has an unlimited timeout setting.